openapi/components/schemas/BankAccounts/AccountNumberTypes/BBANType.yaml

```
@@ -8,55 +8,53 @@ required:
   - routingNumber
   - accountType
   - billingAddress
-allOf:
-  - $ref: ../BankAccountCreatePlain.yaml
-  - properties:
-      method:
-        description: Payment method of the payment instrument.
-        type: string
-        enum:
-          - ach
-      customerId:
-        $ref: ../../CustomerId.yaml
-      accountNumberType:
-        description: |-
-          Bank account number type.
-          A valid value is a Basic Bank Account Number (BBAN) or an International Bank Account Number (IBAN).
-        type: string
-        default: BBAN
-        enum:
-          - IBAN
-          - BBAN
-      accountNumber:
-        description: |-
-          Customer's bank account number.
-          Detailed information on all ISO 13616-compliant national IBAN formats is available in the [SWIFT IBAN Registry](https://www.swift.com/standards/data-standards/iban).
-        type: string
-        pattern: '^[0-9]+$'
-      routingNumber:
-        description: Bank routing number.
-        type: string
-        pattern: '^[0-9]+$'
-      accountType:
-        description: Bank account type.
-        type: string
-        enum:
-          - checking
-          - savings
-          - other
-      bankName:
-        description: Bank name.
-        type: string
-      bic:
-        description: Bank Identifier Code (BIC).
-        type: string
-      billingAddress:
-        description: Customer's billing address.
-        allOf:
-          - $ref: ../../Contact/ContactObject.yaml
-      customFields:
-        $ref: ../../ResourceCustomFields.yaml
-      riskMetadata:
-        $ref: ../../RiskMetadata.yaml
-      useAsBackup:
-        $ref: ../../UseAsBackup.yaml
+properties:
+  method:
+    description: Payment method of the payment instrument.
+    type: string
+    enum:
+      - ach
+  customerId:
+    $ref: ../../CustomerId.yaml
+  accountNumberType:
+    description: |-
+      Bank account number type.
+      A valid value is a Basic Bank Account Number (BBAN) or an International Bank Account Number (IBAN).
+    type: string
+    default: BBAN
+    enum:
+      - IBAN
+      - BBAN
+  accountNumber:
+    description: |-
+      Customer's bank account number.
+      Detailed information on all ISO 13616-compliant national IBAN formats is available in the [SWIFT IBAN Registry](https://www.swift.com/standards/data-standards/iban).
+    type: string
+    pattern: '^[0-9]+$'
+  routingNumber:
+    description: Bank routing number.
+    type: string
+    pattern: '^[0-9]+$'
+  accountType:
+    description: Bank account type.
+    type: string
+    enum:
+      - checking
+      - savings
+      - other
+  bankName:
+    description: Bank name.
```

nitpick: series of nouns
```suggestion
    description: Routing number of the bank.
    type: string
    pattern: '^[0-9]+$'
  accountType:
    description: Type of bank account.
    type: string
    enum:
      - checking
      - savings
      - other
  bankName:
    description: Name of the bank.
```